azure-pipelines.yml

```diff
@@ -0,0 +1,115 @@
+# Mutilated beyond recognition from the example at:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/go
+
+trigger:
+- v2
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight tests
+  branches:
+    include:
+    - v2
+  always: true
+
+strategy:
+  matrix:
+    linux:
+      imageName: ubuntu-16.04
+      gorootDir: /usr/local
+    mac:
+      imageName: macos-10.13
+      gorootDir: /usr/local
+    windows:
+      imageName: windows-2019
+      gorootDir: C:\
+
+pool:
+  vmImage: $(imageName)
+
+variables:
+  GOROOT: $(gorootDir)/go
+  GOPATH: $(system.defaultWorkingDirectory)/gopath
+  GOBIN:  $(GOPATH)/bin
+  modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)'
+  # TODO: Remove once it's enabled by default
+  GO111MODULE: on
```

Comment:
Oh, we're on Go 1.13 now! We should probably remove this before we forget.

Reply:
Oddly, though, `go get`-ting the coverage analysis tools (github.com/axw/gocov/gocov, github.com/AlekSi/gocov-xml) fail without this. The solution is to either define this for the overall environment or prepend it for the `go get` of those tools.

Reply:
Oh, maybe that's because the command is being run outside of a folder with a go.mod file? In other words, outside of a Go module. (Shouldn't the commands be run from this project's folder, though? And this project is a Go module.)

I'm satisfied leaving this here if we determine that the CI commands are being run from outside a Go module, but if they're being run from inside a folder that is a Go module, I want to figure out why this env var is required...

Reply:
Yes, they're run outside of a Go module. If they're run within the project's folder, they'll be automatically added to go.mod and go.sum, which I don't think is proper because these are just tools for our CI and not really dependencies of the project.

Reply:
Roger that. I guess we'll leave it until Go 1.14 when it _should_ be `on` by default.